.gitignore

```diff
@@ -141,4 +141,6 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 
+.vscode/
```

Comment:
Having some issues with loading your website. Try changing the permissions for this, that would be on aws3 website. Here are the instructions. https://s3.console.aws.amazon.com/s3/home. Then for each of the two buckets beginning statuspage:

1. Enter the bucket
2. Select all contents using the ticky-box
3. Select Actions / Make public
4. Confirm 'make public'